language-server/langserver/src/main/java/edu/umn/cs/melt/silver/langserver/CopperSemanticTokenEncoder.java

```diff
@@ -0,0 +1,140 @@
+package edu.umn.cs.melt.silver.langserver;
```

Comment:
Can you add a few more comments here about what this is meant to do? Or is there additional documentation somewhere to explain this? I see some docs for `insert semantic token` but we (eventually) will want some guide on what one needs to do to make an LSP for some other language.